app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetFragment.java

```diff
@@ -91,17 +92,24 @@ public void onAttach(@NonNull final Context context) {
     }
 
     /**
-     * Set the username in navigationHeader.
+     * Set the username and user level (if available) in navigationHeader.
      */
     private void setUserName() {
-        moreProfile.setText(getUserName());
+        BasicKvStore store = new BasicKvStore(this.getContext(), getUserName());
+        String level = store.getString("userLevel","0");
+        if(level.equals("0"))
```

Comment:
Space after `if`, `{` at the end of the line, per code style linked at https://github.com/commons-app/commons-app-documentation/blob/master/android/Code-style.md

Reply:
My apologies for missing these, and thanks for pointing them out; I've fixed them ;)